### PR TITLE
fix: HELM_DIFF_IGNORE_UNKNOWN_FLAGS ignored issue

### PR DESCRIPTION
**Pull request by yxxhero**
fix: https://github.com/databus23/helm-diff/issues/537

### CODE REVIEW

**Comment by yxxhero** [comment]
@mumoshu

File: cmd/upgrade.go [hunk 1]
```diff
@@ -164,13 +165,18 @@ func newChartCommand() *cobra.Command {
 						" --dry-run=server enables the cluster access with helm-get and the lookup template function."
 				)
 
+				cmdFlags := cmd.Flags()
+
+				// see: https://github.com/databus23/helm-diff/issues/537
+				cmdFlags.ParseErrorsWhitelist.UnknownFlags = unknownFlags
```
Comment (mumoshu) [inline cmd/upgrade.go:171]: Wow, good to know! Thanks..

So does this basically mean `DisableFlagParsing: true,` results in `UnknownFlags NOT set by cobra? (Just confirming for learning purpose

Comment (yxxhero) [reply cmd/upgrade.go:171]: Yes, this will disrupt a lot of functionality. I think we only support -- dry run=something. @mumoshu WDYT?

Comment (mumoshu) [reply cmd/upgrade.go:171]: @yxxhero Thanks! Re dry-run, it won't be a good idea to start dropping the backward-compatibility without any prenotices. We are still in the same major ver of helm-diff (3.x). Dropping the compatibility may break downstream user CI/CD pipelines and/or tools.

I think this justifies having more test coverage in future though 🤔

Comment (yxxhero) [reply cmd/upgrade.go:171]: @mumoshu yeah. make sense.

**Review by mumoshu — APPROVED** [review]
@yxxhero Awesome work! Going to merge this and cut a new release asap.

**Comment by yxxhero** [comment]
@mumoshu there are some other issue should be fixed. do you have any good iede for fixing issue that caused by DisableFlagParsing: true?

**Pull request merged**